components/compiler-families/llvm-compilers/SPECS/llvm-compilers.spec

```diff
@@ -0,0 +1,191 @@
+#----------------------------------------------------------------------------bh-
+# This RPM .spec file is part of the OpenHPC project.
+#
+# It may have been modified from the default version supplied by the underlying
+# release package (if available) in order to apply patches, perform customized
+# build/install configurations, and supply additional files to support
+# desired integration conventions.
+#
+#----------------------------------------------------------------------------eh-
+
+%include %{_sourcedir}/OHPC_macros
+
+%global gnuver 6.3.0
+%global cmakemajor 3.8
+%global cmakever 3.8.1
```

Comment:
It's a very good idea. Can one submission cover two things, namely compiler-family and CMake? Otherwise, just to be able to move from place, I'll stick with this inner cmake for a while until cmake appear in OpenHPC repo.